src/assets/scripts/client/ui/TutorialView.js

```diff
@@ -490,33 +477,10 @@ export default class TutorialView {
 
         this.tutorial_step({
             title: 'Approach Clearances, part 2',
-            text: ['You may choose to enter one command at a time, but air traffic controllers usually do multiple. Particularly in approach clearances,',
-                   'they follow an acronym "PTAC" for the four elements of an approach clearance, the "T" and "C" of which',
-                   'stand for "Turn" and "Clearance", both of which we entered separately in this tutorial. Though longer, it is both ',
-                   'easier and more real-world accurate to enter them together, like this: "fh 250 i 28r".'
-            ].join(' '),
-            side: 'left',
-            position: tutorial_position
-        });
-
-        this.tutorial_step({
-            // TODO: the windsock will be moving soon. Update this when that happens
-            title: 'Wind sock',
```

Comment:
We could update this section to explain the information at the top left of the scope

Reply:
Idk lol I felt that the info at the top left was very self-explanatory, and anyways the local time and altimeter don't really affect the simulation. The airport elevation doesn't really provide much either. The wind at the most can be explained, however, a casual player would simply use the runways automatically assigned to the aircraft. while other players would already know, as this kind of sim attracts people with prior knowledge of aviation as it is.

However, if you want me to change it I have no problem either! :)

Reply:
Good point!